javascript/public/index.html

```diff
@@ -9,7 +9,9 @@
       name="description"
       content="UCSB Courses Search"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="apple-touch-icon" sizes="180x180" href="%PUBLIC_URL%/apple-touch-icon.png">
+   <link rel="icon" type="image/png" sizes="32x32" href="%PUBLIC_URL%/favicon-32x32.png">
+   <link rel="icon" type="image/png" sizes="16x16" href="%PUBLIC_URL%/favicon-16x16.png">
```

Comment:
this is a very small issue, but can we fix the tabbing here so that it's consistent with the lines above (i.e. 4 spaces)? Unfortunately, eslint can't check things in the public directory.

Reply:
Good suggestion. Just made the change!